products/compute/api/22-12/stable-endpoints.md

```diff
@@ -38,7 +38,7 @@ Where:
 As a best practice, update your scripts to use the version-specific API endpoints to ensure that your implementation is fully supported.
 For the version-specific APIs, you will have access to the API Reference and Release Notes documentation for changes or updates that may impact you.
 
-When using the version-specific endpoints, you will need to update your automation scripts approximately once-a- year to stay in sync with the product [support lifecycle](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/upgrade/support_lifecycle.html).
+When using the version-specific endpoints, you will need to update your automation scripts approximately once a year to stay in sync with the product [Support lifecycle for connected components](https://docs.prismacloud.io/en/classic/compute-admin-guide/upgrade/support-lifecycle). If you are upgraded to Darwin, see [Support Lifecycle for Connected Components](https://docs.prismacloud.io/en/enterprise-edition/content-collections/runtime-security/upgrade/support-lifecycle).
```

Comment:
@manukumar6 for the 22-12 PCCE, I think we need to point to 
https://docs.prismacloud.io/en/compute-edition/22-12/admin-guide/upgrade/support-lifecycle

the links you shared are for PCEE, where we have the notion of classic and Darwin.
The link I have suggested is appropriate if this .md page is for Compute Edition v 22.12.

Reply:
this is the case for all URLs with /compute/api/version in the file path.

You have correctly identified the URLs for PCEE on App Sec and Compute. Thanks!

Reply:
You're right. These are all Compute/PCCE page. I've updated the links as suggested.